tests/cli/Cli/Task/ConstructCest.php

```diff
@@ -34,8 +36,9 @@ public function _before(CliTester $I)
     public function cliTaskConstruct(CliTester $I)
     {
         $I->wantToTest('Cli\Task - __construct()');
-
-        $I->skipTest('Need implementation');
+        require_once dataDir('fixtures/tasks/MainTask.php');
+        $task = new MainTask();
+        $I->assertInstanceOf(Task::class, $task);
```

Comment:
I have no idea what exactly we're test here. Just a PHP's ability to "require" a file or execute a function called __constructor? And why "require_once"? Shouldn't we use composer autoload policy? IMO would be better to test this
https://github.com/phalcon/cphalcon/blob/762184777a6dc425787ffee531a7b6c8a4b2532d/phalcon/Cli/Task.zep#L46-L48

Reply:
I only moved this from the original test to proper place. onConstruct is tested below